submissions/LGeorge007/memory-pair-game/script.js

```diff
@@ -0,0 +1,65 @@
+const allCards = document.querySelectorAll(".flipper");
+const arrOfCards = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19];
+let unopenedCards = arrOfCards.length;
+let firstCard;
+let secondCard;
+let clikCounter = 0;
+
+const checkCard = function (event) {
+    const checkMatch = function () {
```

Comment:
There is no point for function `checkMatch` to be declared inside function `checkCard`. Either remove the function completely and place it's code inside `checkCard`'s body, or move `checkMatch` outside of `checkCard`'s body. In your case, every time player clicks a card a new instance of `checkMatch` is created in the memory, which is not good for performance.

Reply:
Fixed it.